Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -504,6 +504,14 @@ theorem LinearMap.isUnit_toMatrix'_iff {f : (n → R) →ₗ[R] n → R} : IsUni
 theorem Matrix.isUnit_toLin'_iff {M : Matrix n n R} : IsUnit M.toLin' ↔ IsUnit M :=
   isUnit_map_iff LinearMap.toMatrixAlgEquiv'.symm M
 
+instance : Module (Matrix n n R) (n → R) :=
+  .compHom _ Matrix.toLinAlgEquiv'.toRingHom
+
+instance : SMulCommClass (Matrix n n R) R (n → R) :=
```

Comment:
This should be true more generally for
```suggestion
instance : SMulCommClass (Matrix n n R) S (n → R) :=
```
and
```lean-4
instance : SMulCommClass S (Matrix n n R) (n → R) :=
```

Reply:
What is `S` supposed to be here?

Reply:
Added in #28442.